pages/heroes/[name].tsx

```diff
@@ -40,15 +119,34 @@ export const getStaticPaths: GetStaticPaths = async () => {
 };
 
 export const getStaticProps: GetStaticProps = async (context) => {
-  const heroes = DBSingleton.getInstance().getHeroesMap();
-  const skillsToHeroMap = DBSingleton.getInstance().getSkillsToHeroMap();
+  const db = DBSingleton.getInstance();
+  const heroes = db.getHeroesMap();
+  const skillsToHeroMap = db.getSkillsToHeroMap();
+  const translateSkillsMap = db.getTranslateSkillsMap();
+  const skillsMap = db.getSkillsMap();
+  const threeCostSkillMap = db.get3cSkillsMap();
+  const translateHeroMap = db.getTranslateHeroMap();
+  const translateSoldiersMap = db.getTranslateSoldiersMap();
+  const soldierMap = db.getSoldierMap();
+  const translateEquipmentMap = db.getTranslateEquipmentMap();
+  const translateClassMap = db.getTranslateClassMap();
+  const translateUIMap = db.getTranslateUIMap();
 
   const name = context.params?.name as string;
   const heroData: Hero = heroes[name];
   return {
     props: {
       heroData,
+      skillsMap,
+      soldierMap,
+      translateUIMap,
       skillsToHeroMap,
+      translateHeroMap,
+      translateClassMap,
+      threeCostSkillMap,
+      translateSkillsMap,
+      translateSoldiersMap,
+      translateEquipmentMap,
```

Comment:
I see why you got forced into this. I will think of solutions post merge

Reply:
yeah. Hero page require too much different data. Perhaps i should transform all related data into super object. Just for this page.